integration-cli/docker_cli_build_test.go

```diff
@@ -6596,6 +6596,127 @@ func (s *DockerSuite) TestBuildWorkdirWindowsPath(c *check.C) {
 	}
 }
 
+func (s *DockerSuite) TestBuildLabel(c *check.C) {
+	name := "testbuildlabel"
+	testLabel := "foo"
+
+	_, err := buildImage(name, `
+  FROM `+minimalBaseImage()+`
+  LABEL default foo
+`, false, []string{"--label", testLabel}...)
+
+	if err != nil {
+		c.Fatal("error building image with labels", err)
+	}
+
+	res := inspectFieldJSON(c, name, "Config.Labels")
+
+	var labels map[string]string
+
+	if err := json.Unmarshal([]byte(res), &labels); err != nil {
+		c.Fatal(err)
+	}
+
+	if _, ok := labels[testLabel]; !ok {
+		c.Fatal("label not found in image")
+	}
+}
+
+func (s *DockerSuite) TestBuildLabelCacheCommit(c *check.C) {
```

Comment:
@ehazlett ah ok, I missed the `useCache` boolean argument ; the first one forces docker to not use the cache, and the seconds let's docker use it if required. My question is then : does this test `TestBuildLabelCacheCommit` make the assumption that something is already in the cache ? 
- If true, this means that we need to *fill the cache* first as after each test `DockerSuite.TearDownTest` is ran and it deletes all non-protected images, meaning that the cache is pretty much empty :sweat_smile:.
- If not, I'm not sure I understand the point of the test (as it's basically the same as ˙TestBuildLabel` — maybe I'm missing something though.